### PR TITLE
Fix WebGL example links

### PR DESCRIPTION
**Pull request by mikecat**
### Description

Replaced broken links using TextSS.net without manual verification:

* Replaced `https://github.com/mdn/webgl-examples/` to `https://github.com/mdn/dom-examples/tree/main/webgl-examples/`
* Replaced `EmbedGHLiveSample('webgl-examples/` to `EmbedGHLiveSample('dom-examples/webgl-examples/`
* Replaced `http://mdn.github.io/webgl-examples/` to `https://mdn.github.io/dom-examples/webgl-examples/`

### CODE REVIEW

File: files/zh-cn/web/api/webgl_api/tutorial/using_shaders_to_apply_color_in_webgl/index.md [hunk 1]
```diff
@@ -162,6 +162,6 @@ setColorAttribute(gl, buffers, programInfo);
 
 {{EmbedGHLiveSample('dom-examples/webgl-examples/tutorial/sample3/index.html', 670, 510) }}
 
-[查看完整代码](https://github.com/mdn/webgl-examples/tree/gh-pages/tutorial/sample3) | [在新页面中打开示例](http://mdn.github.io/webgl-examples/tutorial/sample3/)
+[查看完整代码](https://github.com/mdn/dom-examples/tree/main/webgl-examples/tree/gh-pages/tutorial/sample3) | [在新页面中打开示例](https://mdn.github.io/dom-examples/webgl-examples/tutorial/sample3/)
```
Comment (yin1999) [inline files/zh-cn/web/api/webgl_api/tutorial/using_shaders_to_apply_color_in_webgl/index.md:165]:
```suggestion
[查看完整代码](https://github.com/mdn/dom-examples/tree/main/webgl-examples/tutorial/sample3) | [在新页面中打开示例](https://mdn.github.io/dom-examples/webgl-examples/tutorial/sample3/)
```

File: files/zh-tw/web/api/webgl_api/tutorial/adding_2d_content_to_a_webgl_context/index.md [hunk 2]
```diff
@@ -260,9 +260,9 @@ function drawScene(gl, programInfo, buffers) {
 
 接下來，我們讀入正方形的位置，並把它擺在離相機 6 單位遠的位置。然後我們將正方形頂點的 buffer 綁定到 gl 上。最後我們呼叫{{domxref("WebGLRenderingContext.drawArrays()", "drawArrays()")}}函數來渲染物件。
 
-{{EmbedGHLiveSample('webgl-examples/tutorial/sample2/index.html', 670, 510) }}
+{{EmbedGHLiveSample('dom-examples/webgl-examples/tutorial/sample2/index.html', 670, 510) }}
 
-[檢視完整程式碼](https://github.com/mdn/webgl-examples/tree/gh-pages/tutorial/sample2) | [開啟新頁面來檢視結果](http://mdn.github.io/webgl-examples/tutorial/sample2/)
+[檢視完整程式碼](https://github.com/mdn/dom-examples/tree/main/webgl-examples/tree/gh-pages/tutorial/sample2) | [開啟新頁面來檢視結果](https://mdn.github.io/dom-examples/webgl-examples/tutorial/sample2/)
```
Comment (yin1999) [inline files/zh-tw/web/api/webgl_api/tutorial/adding_2d_content_to_a_webgl_context/index.md:265]:
```suggestion
[檢視完整程式碼](https://github.com/mdn/dom-examples/tree/main/webgl-examples/tutorial/sample2) | [開啟新頁面來檢視結果](https://mdn.github.io/dom-examples/webgl-examples/tutorial/sample2/)
```

File: files/zh-tw/web/api/webgl_api/tutorial/getting_started_with_webgl/index.md [hunk 3]
```diff
@@ -9,7 +9,7 @@ slug: Web/API/WebGL_API/Tutorial/Getting_started_with_WebGL
 
 這篇文章將會向您介紹 WebGL 的基礎。這篇文章假設您已經對於三維圖學涉及的數學有所了解，並且它將不會被佯裝為三維圖學的教材。在我們的學習區域內有初學者指南讓你完成編程任務：[Learn WebGL for 2D and 3D graphics](/zh-TW/docs/Learn/WebGL).
 
-在此教學文件中的程式碼範例都能在 [webgl-examples GitHub repository](https://github.com/mdn/webgl-examples/tree/gh-pages/tutorial) 之中找到。
+在此教學文件中的程式碼範例都能在 [webgl-examples GitHub repository](https://github.com/mdn/dom-examples/tree/main/webgl-examples/tree/gh-pages/tutorial) 之中找到。
```
Comment (yin1999) [inline files/zh-tw/web/api/webgl_api/tutorial/getting_started_with_webgl/index.md:12]:
```suggestion
在此教學文件中的程式碼範例都能在 [webgl-examples GitHub repository](https://github.com/mdn/dom-examples/tree/main/webgl-examples/tutorial) 之中找到。
```

File: files/zh-tw/web/api/webgl_api/tutorial/getting_started_with_webgl/index.md [hunk 4]
```diff
@@ -57,9 +57,9 @@ function main() {
 
 至此，您已經有足夠初始化 WebGL 背景資料的程式碼，並且準備好了等待接收內容的容器。
 
-{{EmbedGHLiveSample('webgl-examples/tutorial/sample1/index.html', 670, 510) }}
+{{EmbedGHLiveSample('dom-examples/webgl-examples/tutorial/sample1/index.html', 670, 510) }}
 
-[檢視完整程式碼](https://github.com/mdn/webgl-examples/tree/gh-pages/tutorial/sample1) | [開啟新頁面來檢視結果](http://mdn.github.io/webgl-examples/tutorial/sample1/)
+[檢視完整程式碼](https://github.com/mdn/dom-examples/tree/main/webgl-examples/tree/gh-pages/tutorial/sample1) | [開啟新頁面來檢視結果](https://mdn.github.io/dom-examples/webgl-examples/tutorial/sample1/)
```
Comment (yin1999) [inline files/zh-tw/web/api/webgl_api/tutorial/getting_started_with_webgl/index.md:62]:
```suggestion
[檢視完整程式碼](https://github.com/mdn/dom-examples/tree/main/webgl-examples/tutorial/sample1) | [開啟新頁面來檢視結果](https://mdn.github.io/dom-examples/webgl-examples/tutorial/sample1/)
```

**Review by yin1999 — CHANGES_REQUESTED** [review]
The URLs to source code seem to be wrong. Please fix them

**Comment by mikecat** [comment]
@yin1999 Thank you for pointing out.

New fixes:

* Replaced `github.com/mdn/dom-examples/tree/main/webgl-examples/tree/gh-pages/` to `github.com/mdn/dom-examples/tree/main/webgl-examples/` via TextSS.net
* Manually fixed `https://mdn.github.io/webgl-examples/` URLs to `https://mdn.github.io/dom-examples/`

**Review by Graywolf9 — APPROVED** [review]
Thank you so much! LGTM for `es`

**Review by yin1999 — APPROVED** [review]
Thanks @mikecat. It looks good to me now (for `l10n-zh`)!

**Review by sashasushko — APPROVED** [review]
Approve for `ru`. Thanks 🤝

**Review by SphinxKnight — APPROVED** [review]
Thanks a lot @mikecat! I've pushed a commit (strictly limited to fr) to fix some webgl related links on a page (which needs to be updated) and some videos URLs on another when the examples didn't work anymore.

Edit: OK for `fr` :)

**Review by josielrocha — APPROVED** [review]
LGTM for `pt-BR`. 👍🏼!!

**Comment by yin1999** [comment]
ping @mdn/yari-content-ja :)

**Review by mfuji09 — APPROVED** [review]
LGTM for `ja`.

**Pull request merged**